babel/src/board.js

```diff
@@ -67,15 +67,19 @@ export function iterchunks (orbs, chunkLimitRange = [4, 2]) {
 
 export function findMatches(orbs) {
     let matches = [];
-    let b = orbs;
     let height = orbs.length;
     let width = orbs[0].length;
 
-    _.each(_.range(height - 2), x => {
+    _.each(_.range(height), x => {
         _.each(_.range(width - 2), y => {
-            if (b[x][y] == b[x][y + 1] && b[x][y] == b[x][y + 2]) {
+            if (orbs[x][y] == orbs[x][y + 1] && orbs[x][y] == orbs[x][y + 2]) {
                 matches.push([[x, y], [x, y + 1], [x, y + 2]])
-            } else if (b[x][y] == b[x + 1][y] && b[x][y] == b[x + 2][y]) {
+            };
+        });
+    });
+    _.each(_.range(height - 2), x => {
+        _.each(_.range(width), y => {
+            if (orbs[x][y] == orbs[x + 1][y] && orbs[x][y] == orbs[x + 2][y]) {
```

Comment:
Is there a way to get the coordinates without looping through them with two `_.each` calls? If not, I figured we might as well loop through them in `findMatches` since that's where we need them. But I'm looping through them twice which isn't nice-looking.